dotCMS/src/integration-test/java/com/dotcms/rest/WebResourceIntegrationTest.java

```diff
@@ -342,4 +345,12 @@ public void disallow_anon_when_required_frontend_but_not_backend() throws Except
 
   }
 
+  @Test(expected = InvalidLicenseException.class)
+  public void webResource_NoLicense_RequiredLicense_returnInvalidLicenseException() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)